### PR TITLE
[docs]add advanced-performance-zh-CN.md

### PR DESCRIPTION
**Pull request by hepeguo**
add advanced-performance-zh-CN.md

### CODE REVIEW

File: docs/docs/11-advanced-performance.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+prev: perf-zh-CN.html
+next: context-zh-CN.html
+next: context.html
+---
+
+当大家考虑在项目中使用React的时候，第一个问题往往是他们的应用的速度和响应是否能和非React版一样，每当状态改变的时候就重新渲染组件的整个子树，让大家怀疑这会不会对性能造成负面影响。React用了一些黑科技来减少UI更新需要的花费较大的DOM操作。
+
+## 使用production版本
+
+如果你在你的React app中进行性能测试或在寻找性能问题，一定要确定你在使用[minified production build](/react/downloads.html)。开发者版本包括额外的警告信息对你在开发你的app的时候很有用，但是它比较慢，因为要进行多余处理。
```
Comment (wong2) [inline docs/docs/11-advanced-performance.zh-CN.md]: 开发者版本包括额外的警告信息，这在你开发app的时候很有用，但是因为要进行额外的处理，所以它也会比较慢

File: docs/docs/11-advanced-performance.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+
+```javascript
+shouldComponentUpdate: function(nextProps, nextState) {
+  return true;
+}
+```
+
+一定要记住，React会非常频繁的调用这个函数，所以要确保它的执行速度够快。
+
+假如你有个带有多个聊天线的消息应用，如果只有一条线发生改变，如果我们在`ChatThread`组件执行`shouldComponentUpdate`，React可以跳过其他线的重新渲染步骤。
```
Comment (wong2) [inline docs/docs/11-advanced-performance.zh-CN.md]: 聊天线？？  这里 thread 应该可以翻译为 对话

File: docs/docs/11-advanced-performance.zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+## 使用production版本
+
+如果你在你的React app中进行性能测试或在寻找性能问题，一定要确定你在使用[minified production build](/react/downloads.html)。开发者版本包括额外的警告信息对你在开发你的app的时候很有用，但是它比较慢，因为要进行多余处理。
+
+## 避免更新DOM
+
+React使用虚拟DOM，它是在浏览器中的DOM子树的渲染描述，这个平行的描述让React避免创建和操作DOM节点，这些远比操作一个JavaScript对象慢。当一个组件的props或state改变，React会构造一个新的虚拟DOM和旧的进行对比来决定真实DOM更新的必要性，只有在它们不相等的时候，React才会使用尽量少的改动更新DOM。
+
+在此之上，React提供了生命周期函数，`shouldComponentUpdate`，在重新渲染机制回路（虚拟DOM对比和DOM更新）之前会被触发，给开发者减短这个这个过程回路的能力。这个函数默认返回`true`，让React执行更新。
```
Comment (cht8687) [inline docs/docs/11-advanced-performance.zh-CN.md]: “给开发者减短这个这个过程回路的能力。“  => 赋予开发者跳过这个过程的能力。 verb short-circuit : (transitive) to bypass (a procedure, regulation, etc)

**Comment by gaearon** [comment]
Thanks for the PR!

We just released [new docs mostly written from scratch](https://facebook.github.io/react/). However this means that all old localized content is gone, as new one needs to be written.

We will track this effort in https://github.com/facebook/react/issues/8063. The goal is to make it easier for people to review localizations so you wouldn't have to wait for the core team, and your PRs would be reviewed much faster.

Sorry, and thank you for your work!

**Pull request closed without merging**